eks/examples/cnpack/README.md

```diff
@@ -224,8 +224,9 @@ No requirements.
 | <a name="input_cluster_name"></a> [cluster\_name](#input\_cluster\_name) | Name of the cluster | `string` | n/a | yes |
 | <a name="input_common_name"></a> [common\_name](#input\_common\_name) | Common Name for PCA Creation | `string` | `"cluster.local"` | no |
 | <a name="input_fluentbit_enabled"></a> [fluentbit\_enabled](#input\_fluentbit\_enabled) | Set to true to enable, false to disable | `bool` | `true` | no |
-| <a name="input_metrics_server_enabled"></a> [metrics\_server\_enabled](#input\_metrics\_server\_enabled) | Set to true to enable the network support for Metrics Server, false to disable | `bool` | `false` | no |
+| <a name="input_metrics_server_enabled"></a> [metrics\_server\_enabled](#input\_metrics\_server\_enabled) | Set to true to enable the network support for Metrics Server, false to disable | `bool` | `true` | no |
```

Comment:
Is this a typo fix? I don't see a corresponding change in the code

Reply:
yes a typo fix, the docs are auto generated from `terraform-docs markdown .` and this is probably missed in some previous commit.